languageserver/integration/codelenses.go

```diff
@@ -145,9 +145,11 @@ func (i *FlowIntegration) entryPointActions(
 		return codeLenses, nil
 	}
 
+	emptyEntryPoints := len(entryPointInfo.parameters) == 0
```

Comment:
Maybe rename this to e.g. `noEntryPointParameters` or just `noParameters`, as the current name looks like it implies there are no entry points, but the test is for the parameters

Reply:
Good point :)